src/abTestingInfra/ExperimentContext/index.tsx

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+
+// TODO: maybe we can come up with a better default context value
```

Comment:
+1, what are you thinking? Or alternately, describe this to me :)

Reply:
Frankly, I have no idea what will be better default context value :)